XenAdmin/Alerts/Types/XenCenterUpdateAlert.cs

```diff
@@ -102,15 +102,20 @@ public override string HelpID
         }
 
         public override void Dismiss()
-        {            
-            Properties.Settings.Default.LatestXenCenterSeen = NewVersion.VersionAndLang;
+        {
+            List<string> current = new List<string>(Properties.Settings.Default.LatestXenCenterSeen.Split(','));
+            if (current.Contains(NewVersion.VersionAndLang))
+                return;
+            current.Add(NewVersion.VersionAndLang);

```

Comment:
Thanks @GaborApatiNagy for the suggestion, I've added a new commit, where the size of this list is limited to 5, cleaning up from the begining of the list.